### PR TITLE
ISPN-11145 updates to xsite replication docs

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11145 

This PR updates cross-site replication docs to be more consistent with the main Infinispan xsite docs. Also adds a Tech Preview label to cross-site for the product.

### CODE REVIEW

File: documentation/asciidoc/stories/assembly_xsite_replication.adoc [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-{brandname} allows you to back up data across geographic locations. For
-example, you have a {brandname} cluster running on {openshift} in a data
-center in New York (SiteA). You have another {brandname} cluster running on
-{openshift} in a data center in San Francisco (SiteB).
+Set up cross-site replication to back up data between {brandname} clusters
+running in different locations.
 
-Configuring cross-site replication enables the {brandname} cluster at SiteA to
-back up data to the {brandname} cluster at SiteB.
+For example, you use the {ispn_operator} to manage a {brandname} cluster at a data center in London, **LON**, and another cluster at a data center in New York City, **NYC**. You can add **LON** and **NYC** as backup locations for one another.
```
Comment (galderz) [inline documentation/asciidoc/stories/assembly_xsite_replication.adoc]: This sentence could lead to some confusion. Technically each site has its own operator, so there's no operator that manages multiple sites. Each sites' operator has details to connect to all locations...etc (you already mention this later on).

File: documentation/asciidoc/topics/yaml_annotated/cr_backup_site.yaml [hunk 2]
```diff
@@ -8,9 +8,9 @@ spec:
         expose:
           type: LoadBalancer <3>
       locations: <4>
-      - name: SiteA <5>
```
Comment (galderz) [inline documentation/asciidoc/topics/yaml_annotated/cr_backup_site.yaml]: You forgot to update `.sites.local.name` above?

File: documentation/asciidoc/topics/ref_xsite_crd.adoc [hunk 3]
```diff
@@ -41,7 +27,14 @@ This secret contains different authentication objects, depending on your
 Kubernetes environment.
 ====
 endif::productized[]
+
 //Product
 ifdef::productized[]
-<6> specifies the secret that contains the service account token for the backup site.
+<1> specifies the DataGrid service. {brandname} supports cross-site replication with the DataGrid service only.
+<2> names the local site for a {brandname} cluster.
+<3> specifies `LoadBalancer` as the service that exposes the {openshiftshort} cluster to externally network traffic.
```
Comment (galderz) [inline documentation/asciidoc/topics/ref_xsite_crd.adoc]: The `LoadBalancer` here is used to communicate between the two sites. It is not for handling external traffic.

File: documentation/asciidoc/topics/yaml_annotated/cr_backup_site.yaml (hunk 2 above)
Comment (oraNod) [reply documentation/asciidoc/topics/yaml_annotated/cr_backup_site.yaml]: great catch. fixed.

**Pull request merged**